lib/eslint.js

```diff
@@ -80,6 +80,25 @@ function parseBooleanConfig(string, comment) {
 function parseJsonConfig(string, location, messages) {
     var items = {};
 
+    // Parses a JSON-like comment by the same way as parsing CLI option.
+    try {
+        items = levn.parse("Object", string) || {};
+
+        // Some tests say that it should ignore invalid comments such as `/*eslint no-alert:abc*/`.
+        // Also, commaless notations have invalid severity:
+        //     "no-alert: 2 no-console: 2" --> {"no-alert": "2 no-console:2"}
+        // Should ignore that case as well.
+        if (ConfigOps.isEveryValidSeverify(items)) {
```

Comment:
I realize this if statement shouldn't throw, but should this be outside the try block anyway just to be safe? (If the condition is prefixed with `items &&`, I think it can be done safely.)